src/renderer/ModalsLayer.js

```diff
@@ -28,30 +28,39 @@ const BackDrop: ThemedComponent<{ state: string }> = styled.div.attrs(({ state }
   transition: opacity 200ms cubic-bezier(0.3, 1, 0.5, 0.8);
 `;
 
-const ModalsLayer = ({ visibleModals }: *) => (
-  <Transition
-    in={visibleModals.length > 0}
-    appear
-    mountOnEnter
-    unmountOnExit
-    timeout={{
-      appear: 100,
-      enter: 100,
-      exit: 200,
-    }}
-  >
-    {state => (
-      <BackDrop state={state}>
-        {/* {// Will only render at the end of december
+const ModalsLayer = ({ visibleModals }: *) => {
+  const filteredModals = visibleModals.filter(
+    ({ name, MODAL_SHOW_ONCE }) => !MODAL_SHOW_ONCE || !global.sessionStorage.getItem(name),
+  );
+  filteredModals.forEach(
+    ({ name, MODAL_SHOW_ONCE }) =>
+      MODAL_SHOW_ONCE && global.sessionStorage.setItem(name, +new Date()),
```

Comment:
`Date.now()` is more explicit and prevent casting. (`+new Date()` looks like a typo)

Reply:
done